packages/frontapp/src/app/components/UserList/__snapshots__/SingleUser.spec.tsx.snap

```diff
@@ -47,13 +47,20 @@ exports[`render SingleUserCard component renders correctly, with snapshots 1`] =
   -ms-flex-direction: column;
   flex-direction: column;
   margin-left: 1rem;
-  row-gap: 0.5rem;
 }
 
 .emotion-3>p:first-of-type {
   margin-bottom: 0.5rem;
 }
 
+@media (max-width: 48rem) {
```

Comment:
Why use rem here? Will it still work if rem value changes?

Reply:
That's an auto generated snapshot